src/pages/index.js

```diff
@@ -1,15 +1,31 @@
 import Head from 'next/head'
 
-export default function Home() {
+export default function Home({ children }) {
```

Comment:
Can you put layout component in components/common/ ? So that we can import it when we need it.

Reply:
Thank you for creating \<PageLayout />
That is a good way to organize the layout, which I did not know!